lib/softirq.c

```diff
@@ -72,8 +72,7 @@ int metal_softirq_allocate(int num)
 	int irq_base;
 
 	if ((metal_softirq_avail + num) >= metal_softirq_num) {
-		metal_log(METAL_LOG_ERROR, "No %d available soft irqs.\r\n",
-			  num);
+		ml_err("No %d available soft irqs.\r\n", num);
```

Comment:
Do you need to replace` metal_log(METAL_LOG_ERROR, ...) `by `ml_err `?

Reply:
No. Please, see the new version.